subprojects/build-cache-http/src/integTest/groovy/org/gradle/caching/http/internal/HttpBuildCacheServiceTest.groovy

```diff
@@ -102,7 +102,7 @@ class HttpBuildCacheServiceTest extends Specification {
         then:
         BuildCacheException exception = thrown()
 
-        exception.message == "Received redirect (HTTP 302) to ${server.uri}/redirect/cache/${key.hashCode} when loading entry from '${server.uri}/cache/${key.hashCode}'. Redirects have a performance penalty and are therefore not supported."
+        exception.message == "Received unexpected redirect (HTTP 302) to ${server.uri}/redirect/cache/${key.hashCode} when storing entry to '${server.uri}/cache/${key.hashCode}'. Ensure the configured URL for the remote build cache is correct."
```

Comment:
I'm not sure we should be hard-coding the HTTP status code. This could be either a 302 or a 301 in practice.

Reply:
In fact, let's just remove it altogether. It's not particularly relevant here what the exact status code is.

Reply:
We are not hard coding the status code, we show the actual status code returned for debugging purposes.

Reply:
Here are the statuses we consider a redirect: https://github.com/gradle/gradle/blob/51d5c770364add7db8459a9c526958a7f5ba8bf2/subprojects/build-cache-http/src/main/java/org/gradle/caching/http/internal/HttpBuildCacheService.java#L131-L133

Reply:
Ah. Ok. Disregard.